101-anf-smb-volume/azuredeploy.json

```diff
@@ -0,0 +1,190 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Same location of resource group for all resources"
+      }
+    },
+    "netappAccountName": {
+      "defaultValue": "[concat('anfacc', uniqueString(resourceGroup().id))]",
+      "type": "String",
+      "metadata": {
+        "description": "Name for the Account. The account name must be unique within the subscription"
+      }
+    },
+    "adUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "The value of Active Directory username"
+      }
+    },
+    "adPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The value of Active Directory password"
+      }
+    },
+    "dnsServerIp": {
+      "type": "string",
+      "metadata": {
+        "description": "DNS server IP address"
+      }
+    },
+    "domainName": {
```

Comment:
This attribute needs to renamed to domainFqdn, in Active Directory terms statin domain name only lead us to an ambiguous choice, it can be NetBios domain name or FQDN, and on our case ANF accepts only FQDN so if the customer uses netbios name the provisioning will fail.